### DISCUSSION TITLE
How to remove the Shorts cell in the home feed

### DISCUSSION THREAD

**Discussion by 615e** [PoomSmart/YTABConfig#79] [Q&A]
It literally takes 90% of the screen for 4 useless videos and it shows this cell between every 2 videos making it so irritating to scroll through the home feed
![image](https://github.com/PoomSmart/YTABConfig/assets/68564160/5553b323-cf67-44a4-8894-9cf36d1ecacd)

**Comment by viktorkuts** [comment]
Although this isn't related to A/B config, I've found that by clicking on the three dots for each short and then clicking do not recommend for all cells, the cells will show up less frequently. When you refresh the home, it will disappear. Just make sure to apply the trick as soon as these cells reappear and then refresh. For me, the cells don't even show up anymore, you just have to fight it once a couple of times. It would be great to see if there is an A/B config for a more permanent solution. Hope this helped.
